src/java/fr/paris/lutece/plugins/blog/business/BlogAdminDashboardDAO.java

```diff
@@ -0,0 +1,61 @@
+package fr.paris.lutece.plugins.blog.business;
+
+import fr.paris.lutece.portal.service.plugin.Plugin;
+import fr.paris.lutece.util.sql.DAOUtil;
+
+public final class BlogAdminDashboardDAO implements IBlogAdminDashboardDAO
```

Comment:
Could you please extract the value of id-dashboard into a parameter so that the code can be reused later for creating other dashboards